endpoints_management/control/wsgi.py

```diff
@@ -409,6 +441,19 @@ def _handle_check_response(self, app_info, check_resp, start_response):
         app_info.api_key_valid = api_key_valid
         return error_msg  # the request cannot continue
 
+    def _handle_quota_response(self, app_info, quota_resp, start_response):
+        code, detail = quota_request.convert_response(
+            quota_resp, self._project_id)
+        if code == httplib.OK:
+            return None  # the quota was OK
```

Comment:
should `app_info.response_code` be set to `httplib.OK` here too?

Reply:
Nope; just because the quota is okay doesn't mean the result of the actual request is OK.